joomla/joomla-cms
# added full parameters for codemirror plugin to install sql for mysql,…

added full parameters for codemirror plugin to install sql for mysql, postgres, and azure

currently not all parameters are set on install - this fixes the problem in #6695 
#### Steps to reproduce the issue
- Install Joomla 3.4
- Enable CodeMirror as the editor
- Open any content - notice the "active line" or html element your cursor is in (e.g. a paragraph, h3, etc.) is highlighted in blue - a confusing highlight since it's the same color the browser highlights _selected_ text in. screenshot: http://monosnap.com/image/1VKZ0zawTBTdohBCXpTn1G1j1WXUJr
#### Test instructions

As this is something created during the installation you can not use the patchtester.
The easiest way is to download the full Joomla package with this fix from https://github.com/jneubauer/joomla-cms/archive/codemirrorsql.zip
- Install Joomla as normal.
- Enable CodeMirror as the editor
- Open any content - notice the "active line" or html element your cursor is in (e.g. a paragraph, h3, etc.) is NOT highlighted in blue
